transformations/spellchecker_perturbation/test.json

```diff
@@ -0,0 +1 @@
+{"type": "SpellCheckerPerturbation", "test_cases": [{"class": "SpellCheckerPerturbation", "inputs": {"sentence": "This is one of the sample test cases for the task"}, "outputs": {"sentence": "This is one of the sample test cases for the task"}}, {"class": "SpellCheckerPerturbation", "inputs": {"sentence": "Andrew finally returned the French book to Chris that I bought last week"}, "outputs": {"sentence": "Andrew finally returned the French book to Chris that I bought last week"}}, {"class": "SpellCheckerPerturbation", "inputs": {"sentence": "Sentences with gapping, such as Paul likes coffee and Mary tea, lack an overt predicate to indicate the relation between two or more arguments."}, "outputs": {"sentence": "Sentences wits gapping, such as Paul likest coffee and Mary tea, lack an overt predicate to indicate the relation between two or more arguments."}}, {"class": "SpellCheckerPerturbation", "inputs": {"sentence": "Alice in Wonderland is a 2010 American live-action/animated dark fantasy adventure film"}, "outputs": {"sentence": "Alice in Wonderland is a 2010 American live-action/animated dark fantasy adventure film"}}, {"class": "SpellCheckerPerturbation", "inputs": {"sentence": "Ujjal Dev Dosanjh served as 33rd Premier of British Columbia from 2000 to 2001"}, "outputs": {"sentence": "Ujjal Dev Dosanjh swerved as 33rd Premier of British Columbia from 2000 to 2001"}}, {"class": "SpellCheckerPerturbation", "inputs": {"sentence": "Neuroplasticity is a continuous processing allowing short-term, medium-term, and long-term remodeling of the neuronosynaptic organization."}, "outputs": {"sentence": "Neuroplasticity is a continuous pr\u00e9cising alloying short-term, medium-term, and long-term remodeling of the neuronosynaptic organization."}}]}
```

Comment:
Can you please format this JSON appropriately instead of being on a single line.

Reply:
Sure, will do